### ISSUE TITLE
Polish-Perl-6-First-Steps-Experience: improve https://perl6.org/downloads/

### ISSUE THREAD

**Issue by soundart** [perl6/user-experience#26]
Hi,

i read:

https://perl6.party/post/A-Call-To-Action--Polish-Perl-6-First-Steps-Experience

and would like to give some feedback: 

The short version is: "$PATH setup needs to be documented better"

The long version: 
My first steps were:

Reading https://perl6.org/
clicked on "Download Rakudo Perl 6"
wondered "oh its a build from source"
followed "A quick start for Linux/Mac/BSD/Mingw etc. follows:"

...
make install

So far so good. Then I tried to use it:

# /opt/rakudo-star-2018.04/bin/perl6
#  ok that worked! I the setup a $PATH variable containing /opt/rakudo-star-2018.04/bin
# then I tried to install a perl6 module I wrote some time ago
$ zef
# not found ??

I then read https://rakudo.org/files/star/source linked from https://perl6.org/downloads/ found the 
export PATH.... statement and pasted it into my shell.

echo "export PATH=$(pwd)/install/bin/:$(pwd)/install/share/perl6/site/bin:\$PATH" >> ~/.bashrc
source ~/.bashrc

Still no fun. The --prefix /opt/rakudo-star-2018.04 was not taken into account.

There seem to exists the following directories named 'bin'
...bin/rakudo-star-2018.04/bin
...bin/rakudo-star-2018.04/share/perl6/bin
...bin/rakudo-star-2018.04/share/perl6/site/bin
...bin/rakudo-star-2018.04/share/perl6/vendor/bin

Basically I would expect that the install instructions guide me through the whole process, so that I can just type 'perl6, zef, etc' afterwards in a shell.

Now I need to find a guide howto transfer the build process of my module form panda to zef.....

**Comment by stmuk** [comment]
When you install Rakudo Star it prints instructions at the end telling you exactly what needs to be in your PATH so it already behaves how you want!
